src/screens/SendSynthetic/SendSyntheticAmount.js

```diff
@@ -0,0 +1,382 @@
+    const { symbol, decimals, iconMonoUrl } = this.assetData;
+
+    // balance
+    const balanceFormatted = formatAmount(this.assetBalance);
+    const balanceTitle = `Available ${symbol === PLR ? 'balance' : 'liquidity'}`;
+
+    // value
+    const currentAmount = parseNumericAmount(value);
+    const showNextButton = !submitPressed && !isEmpty(value);
+
+    // value in fiat
+    const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
+    const valueInFiat = currentAmount * getRate(rates, symbol, fiatCurrency);
+    const valueInFiatFormatted = formatFiat(valueInFiat, baseFiatCurrency);
+    const totalInFiat = this.assetBalance * getRate(rates, symbol, fiatCurrency);
+    const totalInFiatFormatted = formatFiat(totalInFiat, baseFiatCurrency);
+
+    // form
+    const icon = `${SDK_PROVIDER}/${iconMonoUrl}?size=3`;
+    const formStructure = generateFormStructure(intentError, this.assetBalance, decimals);
+    const formFields = getAmountFormFields({ icon, currency: symbol, valueInFiatFormatted });
+
+    // submit button
+    const isNextButtonDisabled = inputHasError
+      || parseNumericAmount(value) <= 0
+      || !isOnline
+      || !!intentError;
+    const nextButtonTitle = 'Next';
+
+    return (
+      <ContainerWithHeader
+        headerProps={{ centerItems: [{ title: `Send synthetic ${symbol}` }] }}
+        keyboardAvoidFooter={(
+          <FooterInner>
+            {showNextButton &&
+            <Button
+              disabled={isNextButtonDisabled}
+              small
+              flexRight
+              title={nextButtonTitle}
+              onPress={this.handleFormSubmit}
+            />
+            }
+            {submitPressed && <Spinner width={20} height={20} />}
+          </FooterInner>
+        )}
+        minAvoidHeight={200}
+      >
+        <BackgroundWrapper>
+          <Wrapper regularPadding>
+            <Form
+              ref={node => { this.syntheticsForm = node; }}
+              type={formStructure}
+              options={formFields}
+              value={value}
+              onChange={this.handleFormChange}
+            />
+            <ActionsWrapper>
+              <SendTokenDetails>
+                <Label small>{balanceTitle}</Label>
+                <TextRow>
+                  <SendTokenDetailsValue>
+                    {balanceFormatted} {symbol}
+                  </SendTokenDetailsValue>
+                  <HelperText>{totalInFiatFormatted}</HelperText>
+                </TextRow>
+              </SendTokenDetails>
+              <TouchableOpacity onPress={this.useMaxValue}>
+                <TextLink>Send All</TextLink>
+              </TouchableOpacity>
+            </ActionsWrapper>
+          </Wrapper>
+        </BackgroundWrapper>
+      </ContainerWithHeader>
+    );
+  }
+}
+
+const mapStateToProps = ({
+  assets: { supportedAssets },
+  rates: { data: rates },
+  appSettings: { data: { baseFiatCurrency } },
+  session: { data: { isOnline } },
+}) => ({
```

Comment:
Allow linter detect some issues here:

```suggestion
}: RootReducerState): $Shape<Props> => ({
```

We're also gonna need to add

```js
import { RootReducerState } from 'reducers/rootReducer';
```